### ISSUE TITLE
Possible hip HD failure

### ISSUE THREAD

**Issue by AndPatr** [ADVRHumanoids/awesome_leg#27]
@ntsagarakis, @Lurentz
This evening  I was testing my calibration plugin for the quadrature current model.
After the calibration procedure, I was about to stop the plugins but, while doing so, I accidentally triggered a homing plugin from the gui and this resulted in the leg hitting the screw on top of the sliding plate.
After that, all the control boards are fine (I can operate them without problems; however, the hip joint was subject to some kind of mechanical damage and now turns ok up to some angle, but then looks completely blocked (like if there was something detached blocking it).
Hopefully the HD is fine and there's only something detached in the way of the joint.
@Lurentz, would it be possible to have a look at the joint tomorrow?
Sorry for the inconvenience

**Comment by ntsagarakis** [comment]
Andrea we can look on it tomorrow.  Just to understand, there was basically an impact, which part of the leg hit the screw?
Is the hip joint now completely blocked or you can move within some range?

**Comment by AndPatr** [comment]
The shin of the leg hit the screw, but the problem is on the hip joint. The hip joint looks only partially blocked. 
Moving it by hand, it feels basically normal in the range, let's say, [-pi/2, 0 + some_angle] and then at some_angle is completely stuck.

**Comment by ntsagarakis** [comment]
Do you think that the hip joint may hit its mechanical stop before the shin hit the screw?  May something become loose inside the actuator/link side e.g. the mechanical stop. We will check it tomorrow.

**Comment by AndPatr** [comment]
I am not sure about that. For sure, by moving a bit the leg, it looks like there's something detached which is blocking the hip from rotating.

**Comment by AndPatr** [comment]
@ntsagarakis, @Lurentz I just tested the leg and looks perfectly operational to me.
Thanks a lot for the quick fix. 
I will close this issue

**Comment by ntsagarakis** [comment]
Ok, there is a link encoder error apparently. Today when we inspected the joint there was a sign of small sign of scratch on the PCB of the link encoder, probably from some piece of metal from the broken mechanical stop. But we did not touch the link encoder as we thought that this did not affect the encoder, but probably the encoder head board or line driver of the encoder is not completely healthy. We will check it tomorrow.
